pkg/redshift/api/api.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/grafana/sqlds/v2"
 )
 
+var validStatuses = []string{"SUBMITTED", "PICKED", "STARTED", "FINISHED"}
```

Comment:
you can get these statuses from the `redshiftdataapiservice`, for example: `redshiftdataapiservice.StatementStatusStringSubmitted`